src/index.tsx

```diff
@@ -1,19 +1,45 @@
 import * as React from 'react'
 
+export type LinkResult = {
+  accountId: string
+  job: string
+  userSelectedParameters: {
+    amount?: { value: number; unit: '%' | '$' }
+  }
+}
+
+export type ErrorType =
+  | 'clientError'
+  | 'systemError'
+  | 'userActionRequired'
+  | 'platformError'
+  | 'invalidAccountsConfiguration'
+  | 'invalidUserInput'
+  | 'invalidLinkToken'
+
+export type Error = {
+  type: ErrorType
+  code: string
+  message: string
+}
+
+type EventPayload =
```

Comment:
Nit: I think we can remove this - the payloads are defined on the Link side and once you add {} as a possible type, it allows anything compile time anyway.

Reply:
These types are to help users of our package know the types. Their IDE will pick these type definitions up.